rest-client/src/main/java/org/sagebionetworks/bridge/rest/UserSessionInfoProvider.java

```diff
@@ -24,15 +26,20 @@
 public class UserSessionInfoProvider {
```

Comment:
I actually think, if this has listeners, it could be package private. That way no one will implement against it only to have it changed on them.